docs/releases/v2018.1.2.rst

```diff
@@ -0,0 +1,57 @@
+Gluon 2018.1.2
+==============
+
+Bugfixes
+~~~~~~~~
+
+* Fix a bug leading to missing IPv6 addresses in respondd announcements
+  (`#1523 <https://github.com/freifunk-gluon/gluon/issues/1523>`_)
+
+  The pattern that was used to match addresses from /proc/net/if_inet6
+  did not expect interface indexes growing past two characters.
+
+* Mark ipq806x target as broken for unstable client WiFi
+  (`#1505 <https://github.com/freifunk-gluon/gluon/issues/1505>`_)
+
+  Station connections to the QCA9880 radio on the TP-Link C2600s are frequently
+  disconnected, leading to an abysmal user experience.
+
+* Fix button behaviour on FRITZ!Box 4020
+  (`#1544 <https://github.com/freifunk-gluon/gluon/pull/1544>`_)
+
+  Buttons were triggering an instant reboot into config mode, fix by setting
+  buttons to active low instead of active high.
+
+* Prevent caching of redirects on config mode and status page
+  (`#1530 <https://github.com/freifunk-gluon/gluon/issues/1530>`_)
+
+  As the path to both config mode and status page were changed between versions
+  users could be affected by a redirect to a no more valid URL. 
+
+
+Other changes
+~~~~~~~~~~~~~
+
+* Linux kernel has been updated to v4.4.153
```

Comment:
maybe add something about several bugfixes for batman-adv here, too? 0bb3742f51d1af0ebc1394c72e899524b0adf81d

Reply:
Added a reference to the commit in openwrt-routing, thanks!